third-party/linshare/src/test/java/org/apache/james/linshare/LinshareBlobExportMechanismTest.java

```diff
@@ -110,6 +112,21 @@ void exportShouldSendAnEmailToSharee() throws Exception {
             .body("[1].html", containsString(EXPLANATION));
     }
 
+    @Test
+    void exportShouldShareTheDocumentAndAllowDownloadViaLinShare() throws Exception {
```

Comment:
s/LinShare/Linshare or the other way around. But it should be uniform with the class name.